packages/protocol/contracts/layer1/preconf/impl/PreconfWhitelist.sol

```diff
@@ -14,6 +14,7 @@ contract PreconfWhitelist is EssentialContract, IPreconfWhitelist {
         uint64 activeSince; // Epoch when the operator becomes active.
         uint64 inactiveSince; // Epoch when the operator is no longer active.
         uint8 index; // Index in operatorMapping.
+        string peerIp; // peer ip to connect to for the taiko driver
```

Comment:
Since these need to be conformant to the libp2p `multiAddr`s formed of transport socket address and peer ID, could we rename this field `string multiAddr` for better consistency?

Reply:
sure!